commands/miner.go

```diff
@@ -463,3 +466,51 @@ Values will be output as a ratio where the first number is the miner power and s
 		}),
 	},
 }
+
+type minerListDealResult struct {
```

Comment:
My assumption was that the miner and clients will likely be interested in different aspects of deals in the future, but this may be premature optimization. Do you think I should combine them?